assets/scss/pages/_events.scss

```
@@ -0,0 +1,3 @@
+.card--event__date {
+  font-size: $ic-100;
```

```suggestion
  font-size: $ic-100;
  
  .card {
    &__image {
        height: 275px;

        img {
            transition: .3s ease-out;
        }
    }

    &__body {
        background-color: rgba(255,255,255,.7);
        padding: $ic-200;
        width: 90%;
        margin: 0 auto;
        position: relative;
        top: -40px;
        flex: 1;
        display: flex;
        transition: 0.3s ease-out;
    }

    &__body-inner {
        border: 1px solid #f3f9ff;
        padding: $ic-200;
        flex: 1;
    }
}

.card--event {
    &__date {
        font-size: $ic-100;
        font-weight: 700;
        opacity: .7;
        text-transform: uppercase;
    }

    &:hover, 
    &:focus {
        .card__image img {
            transform: scale(1.05);
        }

        .card__body {
            transform: translateY(-5px);
        }
    }
}
```

Let's update a couple of things here and add a little more styling to the date, add a hover effect so this has some noticeable feedback since it's an interactive element.

This can also instead be moved and placed in the _card.scss file since we already have card styling there.